backend/services/syson-services/src/main/resources/DirectEdit.g4

```diff
@@ -114,26 +107,10 @@ qualifiedName :
 ;
 
 name :
-	BasicName | SingleQuotedString
+	~(':' | ':>' | '::>' | ':>>' | '=' | '[' )+
 ;
 
-BasicName :
-	BasicInitialCharacter BasicNameCharacter*
-;
-
-BasicInitialCharacter :
-	Letter | '_' | '~'
-;
-
-BasicNameCharacter :
-	BasicInitialCharacter | DeciamlDigit
-;
-
-fragment Letter : [a-zA-Z]
-;
-
-fragment DeciamlDigit : [0-9]
-;
+ANY: .;
```

Comment:
Why do you introduce a rule for any character?
Furthermore, It seems not used in the grammar.

Reply:
This is required because the parser rule `name` is now defined as a negation. This means that the parser will match any *token* that isn't one of the negated list. We need to explicitly define a token that matches anything to make sure the negation can match characters that aren't defined in other lexer rules (e.g. special characters).